### ISSUE TITLE
Error: 0x1bc

### ISSUE THREAD

**Issue by Normal-Rowan** [microsoft/WSL2-Linux-Kernel#170]
so i got this error when i ran the command
wsl --set-default-version 2
i have an amd ryzen five the latest version of windows and followed all the steps
please help i would love to have a linux subsystem
![Annotation 2020-08-12 222453](https://user-images.githubusercontent.com/65603480/90070085-32725380-dceb-11ea-814f-d17f67a43e54.png)

**Comment by emacsenli** [comment]
yes I have met same bug with my Ryzen 4700U   notebook
I have enabled wsl2  2 weeks ago and encountered  screen splashing problem ，link as below.
https://github.com/microsoft/WSL2-Linux-Kernel/issues/166
I have to reinstall win10 ,and only enabled wsl for several days since then. today I want to try wsl2  but failed as 01xbc error

all updated patches of my win10 is KB4566782, KB4569745,KB4565627,KB4561600

since I have only run cmd "dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart"  and restart before "wsl --set-default-version 2", but sadly ,screen began to splash as before. So I am wondering there must be some unknown conflicts during restart( I have noticed there is some update while restarting ,isn't? )

**Comment by 14kaitou** [comment]
have you enable the VT in the BIOS and set it to enable but before that check your task manager and see in the performance menu/right bottom menu that your VIRTUALIZATION is enable or not if you still got the same issues I think you've got to do a clean install but before you go that far check your manufacture that your hardware supports the VT or not but I pretty much think is in the OS issue Corrupted system

**Comment by emacsenli** [comment]
yes ,I have checked CPU menu if  performance and  VT is enabled.

**Comment by 14kaitou** [comment]
then it could be that WSL has not supported your GPU yet, hanging tight there @emacsenli

**Comment by Naville** [comment]
Initially ran into the same issue. After rebooted a few times over several days 0xBC error is gone. Weired magic

**Comment by licanhua** [comment]
0xBC means kernel is missing. https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x1bc-when-wsl---set-default-version-2
